packages/api/lib/create-test-data.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const { randomString, randomId } = require('@cumulus/common/test-utils');
+const { s3PutObject } = require('@cumulus/aws-client/S3');
+const {
+  FilePgModel,
+  GranulePgModel,
+  CollectionPgModel,
+} = require('@cumulus/db');
+
+const { constructCollectionId } = require('@cumulus/message/Collections');
+
+// PG mock data factories
```

Comment:
Nit:   Consider renaming the factory method (and following) instead of the comments  if this isn't clear from the code itself.

Reply:
Yeah, I think it makes sense to call the new PG factories stuff like `fakeGranuleRecordFactory` as, _eventually_, they'll be the only factories we care about. It's just in this interim where we have factories for both Dynamo and PG that it's sort of confusing.